weatherman/All_tasks.py

```diff
@@ -0,0 +1,292 @@
+import os
+import sys
+
+class readings():
+    highest_temp = 0
+    lowest_temp = 30
+    max_humidity = 0
+    min_humidity = 100
+    date1 = ''
+    date2 = ''
+    date3 = ''
+    date4 = ''
+
+
+def years(year_no):

```

Comment:
Let's defined a new class here which populated and handles a list of object of `Reading` based on what arguments were passed through the command line. Also, use glob and csv.Dictreader to read the matched files from the directory containing the weatherfiles.